OpenRA.Mods.Common/Widgets/Logic/Editor/LayerSelectorLogic.cs

```diff
@@ -35,11 +37,31 @@ public LayerSelectorLogic(Widget widget, WorldRenderer worldRenderer)
 			layerPreviewTemplate = layerTemplateList.Get<ScrollItemWidget>("LAYERPREVIEW_TEMPLATE");
 
 			IntializeLayerPreview(widget);
+
+			var searchTextField = widget.GetOrNull<TextFieldWidget>("SEARCH_TEXTFIELD");
+			if (searchTextField != null)
+			{
+				searchTextField.OnEnterKey = () =>
+				{
+					var text = searchTextField.Text.ToLowerInvariant();
+					ScrollItemWidget item;
+					if (scrollItems.TryGetValue(text, out item))
+					{
+						layerTemplateList.ScrollToItem(item.ItemKey);
```

Comment:
This doesn't work a lot of the time in TS (e.g. `e1`), scrolling to the bottom of the list instead of the selected actor.